### PR TITLE
Bearer authentication object

### PR DESCRIPTION
**Pull request by pandabear15**
The purpose of a Bearer object is to provide a method to authenticate automatically via a token generated by the server. Tokens are associated to a string that identifies a client, and whenever authentication to access a resource is required, we can call the bearer object to see if the provided token exists and is associated with the client. If so, then access to the resource can be granted.

### CODE REVIEW

File: backend/tests/test_bearer.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
+from backend.src.bearer.Bearer import Bearer
+
+
+def test_add_one_token():
```
Comment (jspaniac) [inline backend/tests/test_bearer.py:4]: Should probably comment comment these

**Review by jspaniac — COMMENTED** [review]
Looks great - question though, how are we planning on getting unique strings for users based on their requests?

File: backend/tests/test_bearer.py (hunk 1 above)
Comment (pandabear15) [reply backend/tests/test_bearer.py:4]: Done :)

**Review by jspaniac — APPROVED** [review]
LGTM!

**Pull request merged**